queries/analytics/event/getEventMetrics.ts

```diff
@@ -46,7 +47,9 @@ async function relationalQuery(
   },
 ) {
   const { toUuid, rawQuery, getDateQuery, getFilterQuery } = prisma;
-  const params: any = [websiteId, startDate, endDate];
+  const website = await getWebsite({ id: websiteId });
```

Comment:
Should probably be 

`const website = await cache.fetchWebsite(websiteId);`

So it fetches from redis if possible.